mitiq/pt/pt.py

```diff
@@ -51,38 +50,28 @@
 ]
 
 
-def execute_with_pauli_twirling(
+def pauli_twirl_circuit(
     circuit: QPROGRAM,
-    executor: Union[Executor, Callable[[QPROGRAM], QuantumResult]],
-    observable: Optional[Observable] = None,
-    *,
     num_circuits: int = 10,
-) -> float:
-    """Estimates the expectation value of the input circuit by averaging
-    expectation values obtained from Pauli twirled circuits.
+) -> List[QPROGRAM]:
+    """Return the Pauli twirled versions of the input circuit.
+
+    Only the $CZ$ and $CNOT$ gates in an input circuit
```

Comment:
Minor nitpick:
```suggestion
    Only the $\mathrm{CZ}$ and $\mathrm{CNOT}$ gates in an input circuit
```

_This line may be too long now, if you agree to the change._

Reply:
Good catch!